src/shared/services/schema-validation.service.ts

```diff
@@ -71,9 +71,27 @@ export class SchemaValidationService {
     '$', 'i'
   );
 
+  private validateDate(value) {
+    let dateFormats = [
```

Comment:
Since this is quite specific of `record-editor`, is it possible to move this customization of the validation to `record-editor` via, e.g some config?